### ISSUE TITLE
Support EC2 Instance Role credentials

### ISSUE THREAD

**Issue by hypnoglow** [hypnoglow/helm-s3#17]
Ref: #15

**Comment by trnubo** [comment]
Hi, I just hit this issue too. I have an EC2 instance configured with an instance IAM role (metadata) but helm-s3 doesn't seem to pickup the credentials.

I can access the s3 bucket with the awscli.

Initially I got this error `2017/12/19 10:26:02 fetch from s3: fetch object from s3: MissingRegion: could not find region configuration` so I added ~/.aws/config with the following

```
[default]
region=ap-southeast-2
```

but then received this error `EmptyStaticCreds: static credentials are empty`

Hopefully helm-s3 will be able to use the EC2 instance role and the default region from ~/.aws/region.

Thanks
